frontend/src/tests/import-export.test.js

```diff
@@ -21,7 +44,7 @@ describe("Import/export library functionality", () => {
         primaryKey: "utorid",
     };
 
-    const instructorData = [
+    const instructorData1 = [
```

Comment:
what is the difference between instructorData1 defined here and the global instructorData?

Reply:
thanks. removed the redundant instructorData1